src/main/java/org/molgenis/data/annotation/makervcf/VcfRecordMapper.java

```diff
@@ -164,22 +169,36 @@ private String createFormatToken(AnnotatedVcfRecord vcfEntity)
 		return stream(formatTokens).collect(joining(":"));
 	}
 
-	private String createSampleToken(VcfSample vcfSample)
-	{
-		String[] sampleTokens = vcfSample.getTokens();
-		return stream(sampleTokens).collect(joining(":"));
-	}
-
-	private String getRlv(GavinRecord gavinRecord, boolean splitRlvField)
+	private String getRlv(GavinRecord gavinRecord, boolean splitRlvField, boolean prefixSplittedFields)
 	{
 		LOG.debug("[MakeRVCFforClinicalVariants] Looking at: {}", gavinRecord);
 
 		List<Relevance> relevance = gavinRecord.getRelevance();
-		String rlv = rlvInfoMapper.map(relevance, splitRlvField);
+		String rlv = rlvInfoMapper.map(relevance, splitRlvField, prefixSplittedFields);
 
 		LOG.debug("[MakeRVCFforClinicalVariants] Converted relevant variant to a VCF INFO field for writing out: {}",
 				rlv);
 
 		return rlv;
 	}
+
+	private String getAnn(GavinRecord gavinRecord)
+	{
+		String result = "";
+		Optional<VcfInfo> annotationInfoField = VcfRecordUtils.getInformation("ANN",
```

Comment:
How about creating a ```private static final String ANN``` for this magic value (or use an existing constant if available)?